modules/microshift-version-api.adoc

```diff
@@ -1,13 +1,15 @@
 // Module included in the following assemblies:
 //
-// * microshift_support/microshift-version.adoc
+// * microshift_architecture/microshift-version-api.adoc
+
+:_content-type: CONCEPT
+[id="microshift-version-api_{context}"]
 
-[id="microshift-version-api"]
 = Checking the {product-title} Version via the API
 
 .Procedure
 
-* Using the API, access the `kube-public/microshift-version` ConfigMap to retrieve the same information.
+* Using an API command, access the `kube-public/microshift-version` ConfigMap to get the version number.
```

Comment:
The phrasing "API command" tripped me up here. Is that how we talk about oc in other docs?